### DISCUSSION TITLE
Theming

### DISCUSSION THREAD

**Discussion by mapsmarketing** [GrapesJS/grapesjs#5413] [Q&A]
I've been looking at the following docs: https://grapesjs.com/docs/getting-started.html#theming

The /src/ directory is not included when you import the files via NPM: https://www.npmjs.com/package/grapesjs?activeTab=code

So, you won't be able to override the theme as described by the docs.

**Comment by artf** [comment]
More about it [here](https://github.com/GrapesJS/grapesjs/issues/5392#issuecomment-1731960080)

**Reply by mapsmarketing** [reply]
Thank you very much @artf
